commit 5db0cd55d4db9cc71f32dc0097e2f014c22967bc
Author:     Andrew Bartlett <abartlet@samba.org>
AuthorDate: Mon May 9 17:43:45 2011 +0200
Commit:     Andrew Bartlett <abartlet@samba.org>
CommitDate: Fri May 13 18:50:23 2011 +0200

    lib/util/ Fix crash bug caused by gfree_debug()
    
    The issue is that we should reset the debug_num_classes to 0 when we
    un-initialise the debug system.
    
    Andrew Bartlett
---
 lib/util/debug.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/util/debug.c b/lib/util/debug.c
index b0a7882..c1b33de 100644
--- a/lib/util/debug.c
+++ b/lib/util/debug.c
@@ -203,7 +203,7 @@ void gfree_debugsyms(void)
 
 	TALLOC_FREE(format_bufr);
 
-	debug_num_classes = DBGC_MAX_FIXED;
+	debug_num_classes = 0;
 
 	state.initialized = false;
 }
